vault/helm/vault/templates/vault-configmap.yaml

```diff
@@ -6,38 +6,71 @@ metadata:
   {{ include "vault-plural.labels" . | nindent 4 }}
 data:
   admin-policy.hcl: |
-    # Allow everything
-    path "*" {
+    # Read system health check
```

Comment:
@DavidSpek Could you please provide me with a rationale as to why you reverted the admin policy to what it was before my change? I based my change on [this](https://discuss.hashicorp.com/t/vault-admin-policy/39803). Many thanks.

Reply:
I can see you modified this:
```hcl
    # List, create, update, and delete key/value secrets
    path "secret/*"
    {
      capabilities = ["create", "read", "update", "delete", "list", "sudo"]
    }
``` 
into this:
```
    # List, create, update, and delete key/value secrets
    path "*"
    {
      capabilities = ["create", "read", "update", "delete", "list", "sudo"]
    }
```
which pretty much opens up everything, except for the things with a more specific path you otherwise restricted in this same policy... The comment on the StackOverflow post is about applying least privilege principle, which is not what we want, we wanted to create a "superuser" policy where the user does have the ability to do everything they want.

Anyway, the policy you used and modified is not too bad, I guess. We can always tweak it later if some users complain.